core/chains/evm/client/client.go

```diff
@@ -197,6 +202,107 @@ func (client *client) HeaderByHash(ctx context.Context, h common.Hash) (*types.H
 	return client.pool.HeaderByHash(ctx, h)
 }
 
+func (client *client) SendTransactionAndReturnErrorType(ctx context.Context, tx *types.Transaction, fromAddress common.Address) (txmtypes.TxmErrorType, error) {
```

Comment:
The method name need not say SendTransactionAndReturnErrorType.
SendTransaction seems good. And since you have a new param added, it won't conflict with previous version.

Reply:
AFAIK operator overloading is not supported in Go

Reply:
yeah go doesn't support function overloading.

this func wraps the separate `SendTransaction(...)` func

Reply:
Ahh, didn't realize. Btw, could we just delete the old method?